lib/plan/setting_panel/setting_panel_state.dart

```diff
@@ -0,0 +1,228 @@
+part of 'setting_panel_cubit.dart';
+
+enum WalkingSpeed { slow, calm, average, prompt, fast }
+enum BikingSpeed { slow, calm, average, prompt, fast }
+enum WalkBoardCost { defaultCost, walkBoardCostHigh }
+enum BikeRentalNetwork { taxi, carSharing, regioRad }
+enum OptimizeType { quick, safe, flat, greenWays, triangle, transfers }
```

Comment:
I would not create all the enums inside of the state, it would be better to move them to a enum file or create for each enum a separated file.

Reply:
Thanks :) , it is changed